scipy/integrate/_bvp.py

```diff
@@ -1097,34 +1108,34 @@ def solve_bvp(fun, bc, x, y, p=None, S=None, fun_jac=None, bc_jac=None,
             status = 1
             if verbose == 2:
                 nodes_added = "({})".format(nodes_added)
-                print_iteration_progress(iteration, max_rms_res, m,
+                print_iteration_progress(iteration, max_res, m,
```

Comment:
In `print_iteration_progress` I suggest that we should report maximum boundary condition residuals.

Reply:
`print_iteration_header()` and `print_iteration_progress()` have been modified to report `max_bc_res`